test/models/observation_test.rb

```diff
@@ -95,10 +125,23 @@ def test_specimens
     assert(observations(:detailed_unknown).specimens.length > 0)
   end
 
+  def test_observer_takes_general_email_questions_from_viewer
+    user_a = users(:rolf) # rolf takes general email questions
```

Comment:
Will do 1 assertion/test and rewrite.  For instance as:
```ruby
  def test_observer_accepts_general_email_questions
    obs_by_user_taking_questions = observations(:coprinus_comatus_obs)
    assert(obs_by_user_taking_questions.
             observer_takes_email_questions_from?(users(:dick)),
           "User with email_general_question should take questions from others")
  end
```
I hope that's also somewhat more intuitive.
But it's arbitrary that the owner of the observation accepts general emails.
I think the only way to make it completely obvious would be some db queries setting up the assertion. 
Something like (pseudocode):
```ruby
obs = Observation.where(email_general_question: true)
viewer = User.where.not(obs.user.id)
assert(obs.observer_takes_email_questions_from?(viewer),
           "User with email_general_question should take questions from others")
```
But I don't want to hit the db in what should be a unit test.